pmlc/dialect/stripe/dialect.h

```diff
@@ -27,7 +23,7 @@ class Dialect : public mlir::Dialect {
 
   mlir::Type parseTensor(llvm::StringRef tyData, mlir::Location loc) const;
   mlir::Type parseTensorRef(llvm::StringRef tyData, mlir::Location loc) const;
-  mlir::LogicalResult parseTensorSize(llvm::StringRef sizeSpec, mlir::Location loc,    // NOLINT
+  mlir::LogicalResult parseTensorSize(llvm::StringRef sizeSpec, mlir::Location loc,
                                       llvm::SmallVectorImpl<TensorDim>& odims) const;  // NOLINT
```

Comment:
The `pmlc/CPPLINT.cfg` file should allow non-const runtime references so the NOLINT should be unnecessary now.